ferretdb_packaging/packaging-entrypoint.sh

```diff
@@ -10,6 +12,9 @@ sed -i '/internal/d' Makefile
 # Build the Debian package
 debuild -us -uc
 
+# Rename .deb files to include the OS name prefix
+for f in ../*.deb; do mv $f $(echo $f | sed "s/\(.*\)\//\1\/$OS-/g"); done
```

Comment:
Can we do it in some other way? On the one hand, we avoid bash and write a tool in Go to be able not only to write it but read it. On another, we add this peak bash one-liner.

Reply:
Thanks :pray: , see the updated bash. I think it's simple enough for not needing go code.